### PR TITLE
Removes shitty shoehorned references.

### PR DESCRIPTION
**Pull request by Iamgoofball**

### CODE REVIEW

**Comment by Anonmare** [comment]
Is this the PR to finally remove Ian, that shitty Cowboy Bebop reference?

**Comment by InsaneHyena** [comment]
Undertale sucks, good PR.

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -567,15 +567,6 @@
...
 
-	var/list/papyrus_messages = list(
-		"That's the disposal bin. Feel free to visit it at any time.",
-		"I can't just let anyone ERP with me, I'm a skeleton with standards!",
-		"Sorry. Can't talk. I'm busy being popular on a video game.",
-		"You can't spell 'robust' without several letters from my name!!!")
-	var/list/sans_messages = list(
-		"You feel your redtext crawling on your back.",
-		"On days like these, references like this should be BURNING IN HELL.",
-		"I've gotten a ton of work done today. A skele-ton.")
```
Comment (Supermichael777) [inline code/game/objects/items/toys.dm]: this one can go in the normal pile. its jeneric enough that it fits.

**Comment by ChangelingRain** [comment]
You should add more jokes, instead.

File: code/game/objects/items/toys.dm (hunk 1 above)
Comment (coiax) [inline code/game/objects/items/toys.dm]: Spoiler, some of the regular lines are undertale references, but don't tell Goof, he'll go crazy trying to work out which ones they are.

Comment (Iamgoofball) [inline code/game/objects/items/toys.dm]: if they're generic enough to pass as a normal skeleton line they can stay

**Comment by Niknakflak** [comment]
Ian is errorage's actual corgi, not cowboy bebop

**Comment by Incoming5643** [comment]
Doesn't go far enough

**Comment by Bawhoppen** [comment]
im getting so confused what happened to the other pr

**Comment by Incoming5643** [comment]
It was merged despite everyone hating it

**Comment by Man-Shroom-** [comment]
undertale is bad game

**Comment by coiax** [comment]
Doesn't actually remove the undertale references. :-1: Nice meme PR

**Comment by coiax** [comment]
> forced references

With a speech engine inspired by Undertale, with a style of speech inspired by Undertale, in a game which already makes skeletons reference Undertale everytime they talk. Sure Jan.

Removing 8 out of 32 lines of dialogue in some vain attempt at spiting me isn't going to cut it. If you actually care about references, remove the whole thing. Unless this is just some childish swipe, which I'M SURE IT'S NOT.

File: code/game/objects/items/toys.dm [hunk 2]
```diff
@@ -567,15 +567,6 @@
 	attack_verb = list("boned", "dunked on", "worked down to the bone")
```
Comment (Man-Shroom-) [inline code/game/objects/items/toys.dm]: This attack_verb list is a ref

**Comment by PKPenguin321** [comment]
coiax don't act like this is some kind of subtle reference
i mean come on

**Comment by Incoming5643** [comment]
Just because you see a reference to a game is not an invitation to try and jam more references to that game in. Indeed as I've said before it's pretty much the opposite.

File: code/game/objects/items/toys.dm [hunk 3]
```diff
@@ -628,25 +619,17 @@
...
-	toy_talk(user, "You feel like you're going to have a bad time.")
+	toy_talk(user, "RATTLE ME BONES")
 
 	user.Stun(5)
 	sleep(20)
 	return OXYLOSS
 
 /obj/item/toy/talking/skeleton/generate_messages()
-	return list(pick(papyrus_messages + sans_messages + regular_messages))
+	return list(regular_messages)
```
Comment (optimumtact) [inline code/game/objects/items/toys.dm]: GOOF YOU LEFT OUT THE PICK YOU SHITBIRD

Comment (coiax) [inline code/game/objects/items/toys.dm]: TESTING IS FOR OTHER PEOPLE, GOOF IS A BUSY MAN CONTRIBUTING HIGH QUALITY CHANGES

Comment (PKPenguin321) [inline code/game/objects/items/toys.dm]: This wouldn't have happened if we hadn't merged skeleton toys in the first place though coiax 👌

Comment (Iamgoofball) [inline code/game/objects/items/toys.dm]: AAAAAAAAAaaaaa

**Pull request merged**